.gitlab/benchmarks/provisions/user_data/setup-benchmarking-platform-module.ps1

```diff
@@ -0,0 +1,77 @@
+<powershell>
```

Comment:
Taking out the provisions from the tooling repo is a good idea! However let's rather include it in the BP repo as the benchmarking inner logic should live there!